rpi-dp/leaveprob-dttor/plantcondition_recognition.py

```diff
@@ -0,0 +1,30 @@
+import argparse
+
+from keras.preprocessing.image import img_to_array
+from keras.models import load_model
+import numpy as np
+import imutils
+import cv2
+
+import time
+import picamera
+
+def planttp_rec(image, model_path):
```

Comment:
Rename this function to `plantcond_rec`.

Reply:
ok